derivation/src/fpvm_evm/precompiles/test_utils.rs

```diff
@@ -0,0 +1,96 @@
+//! Test utilities for accelerated precompiles.
+#[cfg(test)]
+pub mod test {
```

Comment:
By setting `#[cfg(test)]` to `mod test_utils` in `precompiles/mod.rs`, we can remove this module hierarchy.